src/Components/CreateImageWizardV2/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -57,7 +68,29 @@ export const ImageOutputList = () => {
   const arch = useAppSelector((state) => selectArchitecture(state));
   return (
     <TextContent>
-      <ReleaseLifecycle />
+      {distribution === RHEL_8 && (
+        <>
+          <Text className="pf-v5-u-font-size-sm">
+            {RELEASES.get(distribution)} will be supported through{' '}
+            {toMonthAndYear(RHEL_8_FULL_SUPPORT[0])}, with optional ELS support
+            through {toMonthAndYear(RHEL_8_MAINTENANCE_SUPPORT[0])}. Consider
+            building an image with {RELEASES.get(RHEL_9)} to extend the support
+            period.
+          </Text>
+          <FormGroup label="Release lifecycle">
+            <Panel>
+              <PanelMain maxHeight="10rem">
+                <Bar
```

Comment:
This code is shared with the `<MajorReleasesLifecyclesChart>` component and can also have DRY principles applied.